### PR TITLE
update database migrations document

### PR DESCRIPTION
**Pull request by philipwritescode**
Wanted these docs to be up to date with https://github.com/department-of-veterans-affairs/vets-api/pull/5430

### CODE REVIEW

**Review by ericboehs — COMMENTED** [review]
Thoughts on removing the docs and just referring to the docs in strong_migrations?

**Comment by erluti** [comment]
> Thoughts on removing the docs and just referring to the docs in strong_migrations?

I like that idea in general, but looking though the docs here they seem to be written specifically for vets-api and don't overlap with the instructions in `strong_migrations`

**Review by LindseySaari — APPROVED** [review]
Yay for doc updates 👏

File: platform/engineering/backend/database-migrations.md [hunk 1]
```diff
@@ -62,6 +62,10 @@ Schema update operations involve a multistep process:
   1. Deploy application to production
   1. Perform additional data migration with independent rake task
 
+##### Before making schema updates
+
+Review the checks from `strong_migrations` [here](https://github.com/ankane/strong_migrations#checks) before making your changes.  Below are specific instructions for making certain kinds of changes within `vets-api`.
```
Comment (ericboehs) [inline platform/engineering/backend/database-migrations.md:67]: Two spaces. Ewww.

**Review by ericboehs — APPROVED** [review]

File: platform/engineering/backend/database-migrations.md (hunk 1 above)
Comment (erluti) [reply platform/engineering/backend/database-migrations.md]: it's habit that's been very challenging to break

**Pull request merged**